kinder/cmd/kinder/get/nodes/nodes.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodes
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubeadm/kinder/pkg/cluster/status"
+	"k8s.io/kubeadm/kinder/pkg/constants"
+)
+
+type flagpole struct {
+	Name string
+}
+
+// NewCommand returns a new cobra.Command for getting the list of nodes in a cluster
+func NewCommand() *cobra.Command {
+	flags := &flagpole{}
+
+	cmd := &cobra.Command{
+		Args:  cobra.NoArgs,
+		Use:   "nodes",
+		Short: "lists existing nodes in kind clusters by their name",
```

Comment:
lists --> Lists

Reply:
Sure. Taking care of it. Just FYI, I took the opportunity to also address the same on `nodes/nodes.go`.